### PR TITLE
✨ Integrate locust dashboards

### PR DESCRIPTION
**Pull request by bisgaard-itis**
## What do these changes do?
- Integrate locust-plugins' [dashboards](https://github.com/SvenskaSpel/locust-plugins/tree/master/locust_plugins/dashboards) with our load testing framework (as demoed in the review meeting)
- These dashboard allow users to view (in real time) statistics based on our locust load tests.
- I also did a minor clean up of make recipes and unified settings which were still not handled by env vars

image
image

## Related issue/s

- #5953 

## How to test

See [README](https://github.com/ITISFoundation/osparc-simcore/blob/ea8da8294615c7ecdcdc3436a94e2da7fcf52525/tests/performance/README.md) for how to run load tests and visualize

## Dev-ops checklist

- [x] No ENV changes or I properly updated ENV ([read the instruction](https://git.speag.com/oSparc/osparc-ops-deployment-configuration/-/blob/configs/README.md?ref_type=heads#how-to-update-env-variables))

### CODE REVIEW

File: tests/performance/Makefile [hunk 1]
```diff
@@ -36,14 +41,32 @@ down: ## stops and removes osparc locust containers
 	docker compose --file docker-compose.yml down
 
 .PHONY: test
-test: ## runs osparc locust. locust and test confiuration are specified in .env file next to target file
-	@$(call check_defined, target, please define target file when calling $@ - e.g. ```make $@ target=MY_LOCUST_FILE.py```)
-	@export LOCUST_FILE=$(target); \
-	export ENV_FILE=$$(realpath .env); \
-	if [ ! -f $${ENV_FILE} ]; then echo "You must generate a .env file before running tests!!! See the README..." && exit 1; fi; \
+test: ## runs osparc locust. Locust and test configuration are specified in ENV_FILE
```
Comment (pcrespov) [inline tests/performance/Makefile:44]: Q: does CI starts also this extra service?

**Review by pcrespov — APPROVED** [review]
Good bless MB! ;-)

**Review by matusdrobuliak66 — APPROVED** [review]
cool, looking forward to try it myself

**Review by GitHK — APPROVED** [review]

File: tests/performance/README.md [hunk 2]
```diff
@@ -8,12 +8,22 @@ In the [locust_files] folder are located the test files.
...
 ```
-make test target=locust_files/platform_ping_test.py
+make test
 ```
+
+## Dashboards for visualization
+- You can visualize the results of your tests (in real time) in a collection of beautiful [Grafana dashboards](https://github.com/SvenskaSpel/locust-plugins/tree/master/locust_plugins/dashboards).
+- To do this, run `make dashboards-up` and go to `localhost:3000` to view the dashboards. The way you tell locust to send test results to the database/grafana is by ensuring `LOCUST_TIMESCALE=1` (see how to generate settings in [usage](#usage))
+- When you are done you run `make dashboards-down` to clean up.
+- If you are using VPN you will need to forward port 300 to your local machine to view the dashboard.
```
Comment (sanderegg) [inline tests/performance/README.md:25]: port 300?? that is legal?

**Pull request merged**